static/src/stylesheets/head.content.scss

```diff
@@ -38,3 +38,4 @@
    ========================================================================== */
 
 @import 'module/atoms/_quiz';
+@import 'module/atoms/_youtube';
```

Comment:
@SiAdcock @NataliaLKB - looking for your frontend guidance here! Does this seem like the correct place to import these styles?